pyproject.toml

```diff
@@ -22,6 +22,7 @@ openai = {version ="^0.27.6", optional = true}
 nltk = {version ="^3.8.1", optional = true}
 sentence-transformers = {version ="^2.2.2", optional = true}
 evaluate = {version = "^0.4.0", optional = true}
+faiss-cpu = {version = "^1.7.1", optional = true}
```

Comment:
Presumably this also works with the GPU version? should we add:
`faiss-gpu = {version = "^1.7.1", optional = true}`
so that either would satisfy the optional dependency?